packages/itwin/tree-widget/src/components/trees/external-sources-tree/ExternalSourcesTree.tsx

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import "../VisibilityTreeBase.scss";
+import { ControlledTree, DelayLoadedTreeNodeItem, SelectionMode, useTreeModel } from "@itwin/components-react";
+import { IModelConnection } from "@itwin/core-frontend";
+import { Node, Ruleset } from "@itwin/presentation-common";
+import { usePresentationTreeNodeLoader, useUnifiedSelectionTreeEventHandler } from "@itwin/presentation-components";
+import * as RULESET_EXTERNAL_SOURCES_IMPORT from "./ExternalSources.json";
+
+/**
+ * Presentation rules used by ControlledCategoriesTree
+ * @internal
+ */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
I guess this can be removed as `import` is used instead of `require`

Reply:
yes, cleaned up